sktime/classification/base.py

```diff
@@ -1,12 +1,41 @@
 # -*- coding: utf-8 -*-
-__all__ = ["BaseClassifier"]
+__all__ = [
+    "BaseClassifier",
+    "classifier_list",
+]
 __author__ = ["Markus Löning"]
 
 import numpy as np
 
 from sktime.base import BaseEstimator
 from sktime.utils.validation.panel import check_X
 
+"""
+Main list of classifiers extending this class. For clarity, some utility classifiers,
+such as Proximity Stump, are not listed.
+"""
+classifier_list = [
```

Comment:
you should be able to get a similar list via `all_estimators("classifier")` which crawls the package and returns all classifiers, what's this needed for?

Reply:
because I dont want to list list all the estimators, only the ones I am working with, and it helps me to see them all in a list in the text. If you dont want it in base I will move it